src/cs-graphics/ToneMappingNode.hpp

```diff
@@ -0,0 +1,109 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//                               This file is part of CosmoScout VR                               //
+//      and may be used under the terms of the MIT license. See the LICENSE file for details.     //
+//                        Copyright: (c) 2019 German Aerospace Center (DLR)                       //
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef CS_GRAPHICS_TONEMAPPING_NODE_HPP
+#define CS_GRAPHICS_TONEMAPPING_NODE_HPP
+
+#include "cs_graphics_export.hpp"
+
+#include "HDRBuffer.hpp"
+
+#include <VistaKernel/EventManager/VistaEventHandler.h>
+#include <VistaKernel/GraphicsManager/VistaOpenGLDraw.h>
+#include <memory>
+
+class VistaGLSLShader;
+class IVistaClusterDataCollect;
+class IVistaClusterDataSync;
+
+namespace cs::graphics {
+
+/// The ToneMappingNode is used to bring the content of the HDRBuffer to the LDR backbuffer. It
```

Comment:
Maybe transform instead of bring?
```suggestion
/// The ToneMappingNode is used to transform the content of the HDRBuffer to the LDR backbuffer. It
```